appengine-java8/tasks/README.md

```diff
@@ -79,23 +73,19 @@ Create a task, targeted at the `/tasks/create` endpoint, with a payload specifie
 
 ```
 mvn exec:java -Dexec.mainClass="com.example.task.CreateTask" \
-    -Dexec.args="--project-id $GOOGLE_CLOUD_PROJECT \
+    -Dexec.args="--project-id $PROJECT_ID \
```

Comment:
Why is this being changed? It's GOOGLE_CLOUD_PROJECT in all the samples I can think of, though I wouldn't be surprised if there were inconsistencies. Was there guidance on this that I missed?

Reply:
Trying to make it consistent between the samples; there are many inconsistencies between repos/samples. I don't think there is an official call.